contributors.md

```diff
@@ -292,12 +307,14 @@ The following people are not part of the development team, but have been contrib
 * (ThatGuyYouKnow)
 
 ## RollerCoaster Tycoon 2 credits
-Design and programming by Chris Sawyer
-Graphics by Simon Foster
-Sound and music by Allister Brimble
-Additional sounds recorded by David Ellis
-Representation by Jacqui Lyons at Marjacq Ltd.
+* Allister Brimble - Music & Sound
+* Chris Sawyer - Programming
+* David Ellis - Additional Sound
+* Jacqui Lyons (Marjacq Ltd.) - Representation
+* Simon Foster - Graphics
 
 Thanks to: Peter James Adcock, Joe Booth, and John Wardley
 
-Licensed to Infogrames Interactive Inc.
+Licensed to Atari
```

Comment:
Let’s revert this - the company was called Infogrames Interactive Inc. back in the day. And just "Atari" is not the legal name of the current company anyway.